frontend/src/container/CreateAlertRule/styles.ts

```diff
@@ -0,0 +1,9 @@
+import { Row } from 'antd';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+export const PanelContainer = styled(Row)`
```

Comment:
wrap=false is there probably there is no need for styled components

Reply:
done